### PR TITLE
fix when bundle that is focused on is deleted, the app does not freakout

### PR DESCRIPTION
**Pull request by pbhatnagar3**
#### selenium  tests
all pass
###### with worksheet item

^ we delete the only item in the table and there is no error

^ nothing in the console freaks out

###### with multiple items

@percyliang @kashizui please review.

### CODE REVIEW

File: codalab/apps/web/static/js/worksheet/worksheet_side_panel.js [hunk 1]
```diff
@@ -34,9 +34,11 @@ var WorksheetSidePanel = React.createClass({
 
     getFocus: function() {
         // Return the state to show on the side panel
-        if (this.props.focusIndex == -1)
+        var focussedBunble = this.props.ws.state.items[this.props.focusIndex]
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/worksheet_side_panel.js]: typo

**Comment by percyliang** [comment]
Other than typo, LGTM

File: codalab/apps/web/static/js/worksheet/worksheet_side_panel.js (hunk 1 above)
Comment (pbhatnagar3) [inline codalab/apps/web/static/js/worksheet/worksheet_side_panel.js]: oh. yeah my bad. Sorry my British influenced English does this now and then. I did not know that it is spelled as focused in US. Thanks for letting me know. :+1:

Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/worksheet_side_panel.js]: Well, it's not just that....bundle is misspelled.

**Pull request merged**